documentation/markdown/usage/notifications.md

```diff
@@ -0,0 +1,188 @@
+
+```json
+{
+  "@context": [
+    "https://www.w3.org/ns/activitystreams",
+    "https://www.w3.org/ns/solid/notification/v1"
+  ],
+  "id": "urn:123456:http://example.com/foo",
+  "type": "Update",
+  "object": "http://localhost:3000/foo",
+  "state": "987654",
+  "published": "2023-02-09T15:08:12.345Z"
+}
+```
+
+A notification contains the following fields:
+
+* **id**: A unique identifier for this notification.
+* **type**: What happened to the notification. We discuss the possible values below.
+* **object**: The resource that changed.
+* **state**: An identifier indicating the state of the resource.
+              This corresponds to the ETag value you get when doing a request on the resource itself.
+* **published**: When this change occurred.
+
+### Notification types
+
+CSS supports 5 different notification types that the client can receive.
+The format of the notification can slightly change depending on the type.
+
+Resource notification types:
+
+* **Create**: When the resource is created.
+* **Update**: When the existing resource is changed.
+* **Delete**: When the resource is deleted. Does not have a `state` field.
+
+Additionally, when listening to a container,
+there are 2 extra notifications that are sent out when the contents of the container change.
+For these notifications, the `object` fields references the resource that was added or removed,
+while the new `target` field references the container itself.
+
+* **Add**: When a new resource is added to the container.
+* **Remove**: When a resource is removed from the container.
+
+## Features
+
+The Solid notification specification describes several extra features that can be supported by notification channels.
+By default, these are all supported on the channels of a CSS instance,
+as can be seen in the descriptions returned by the server above.
+Each feature can be enabled by adding a field to the JSON-LD you send during subscription.
+The available fields are:
+
+* **startAt**: An `xsd:dateTime` describing when you want to notifications to start.
+                 No notifications will be sent on this channel before this time.
+* **endAt**: An `xsd:dateTime` describing when the notifications should stop.
+               The channel will be destroyed at that time and no more notifications will be sent.
+* **state**: A string corresponding to the `state` string of a resource notification.
+               In case this value differs from the actual state of the resource,
+               a notification will be sent out immediately to inform the client that its stored state is outdated.
+* **rate**: An `xsd:duration` indicating how often notifications can be sent out.
+              A new notification will only be sent out after this much time passed since the previous notification.
+* **accept**: A description of the content-type in which the client would want to receive the notifications.
+                Expects the same values as an `Accept` HTTP header.
```

Comment:
```suggestion
* **`startAt`**: An `xsd:dateTime` describing when you want notifications to start.
                 No notifications will be sent on this channel before this time.
* **`endAt`**: An `xsd:dateTime` describing when you want notifications to stop.
               The channel will be destroyed at that time, and no more notifications will be sent.
* **`state`**: A string corresponding to the `state` string of a resource notification.
               If this value differs from the actual state of the resource,
               a notification will be sent out immediately to inform the client that its stored state is outdated.
* **`rate`**: An `xsd:duration` indicating how often notifications can be sent out.
              A new notification will only be sent out after this much time has passed since the previous notification.
* **`accept`**: A description of the `content-type(s)` in which the client would want to receive the notifications.
                Expects the same values as an `Accept` HTTP header.
```